test/com/google/enterprise/adaptor/DocumentHandlerTest.java

```diff
@@ -735,6 +753,72 @@ public void getDocContent(Request request, Response response)
   }
 
   @Test
+  public void testMetadataTransformWithLastModified() throws Exception {
+    final Date oDate = new Date(0);
+    final Date nDate = new Date();
+    MockAdaptor adaptor = new MockAdaptor() {
+      @Override
+      public void getDocContent(Request request, Response response)
+          throws IOException, InterruptedException {
+        response.setLastModified(oDate);
+        response.respondNoContent();
+      }
+    };
+    String remoteIp = ex.getRemoteAddress().getAddress().getHostAddress();
+    DocumentHandler handler = createHandlerBuilder()
+        .setAdaptor(adaptor)
+        .setFullAccessHosts(new String[]{remoteIp, "someUnknownHost!@#$"})
+        .setSendDocControls(true)
+        .setGsaVersion("7.4.0-0")
+        .setTransform(new TransformPipeline(Lists.newArrayList(new DocumentTransform() {
+          @Override
+          public void transform(Metadata metadata, Map<String, String> params) {
+            assertEquals(oDate.getTime(), Long.parseLong(params.get("Last-Modified-Millis-UTC")));
```

Comment:
files that are previously 80 char maintain that limit.  new files can use 100 char limit, per style guide. i think this file is historically 80 char and so should maintain being 80 char per line.